java-frontend/src/main/java/org/sonar/java/JavaSquid.java

```diff
@@ -109,18 +109,18 @@ private static VisitorsBridge createVisitorBridge(
   }
 
 
-  public void scan(Iterable<File> sourceFiles, Iterable<File> testFiles) {
+  public void scan(Collection<File> sourceFiles, Collection<File> testFiles) {
```

Comment:
Why this change ?

Reply:
In order to translate more easily the collections of `File` into collections of `String`, using streams, at the bottom of the chain.